src/dmd/backend/dwarfdbginf.d

```diff
@@ -1132,8 +1134,8 @@ static if (ELFOBJ)
 version (MARS)
 {
     debug_info.buf.write("Digital Mars D ");
-    //debug_info.buf.writeString(global._version);     // DW_AT_producer
-    debug_info.buf.writeString("v2.083");              // DW_AT_producer
+    debug_info.buf.write(global._version, 8 /* v2.0XX.X */);     // DW_AT_producer
```

Comment:
`global._version` will contain whatever `git describe` returns. A length of `8` is not enough, it can contain: `v2.084.0-389-geb6890bf5-dirty`, for example.